src/main/java/core/basesyntax/Figure.java

```diff
@@ -0,0 +1,19 @@
+package core.basesyntax;
+
+public abstract class Figure {
+    private String color;
+
+    public Figure(String color) {
+        this.color = color;
+    }
+
+    public void setColor(String color) {
+        this.color = color;
+    }
+
+    public String getColor() {
+        return color;
+    }
+
+    public abstract void draw();
```

Comment:
why did we create a method draw() in the class Figure?

Reply:
Becoes somebody can forget to realize  this method.  If we have abstract method draw(), we must realized it in inherit classes.